.github/workflows/kibblepi_ci_test_main.yml

```diff
@@ -27,4 +27,4 @@ jobs:
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Kibblepi - Test with pytest
       run: |
-        pytest
+        python -m pytest
```

Comment:
@Saxamos  I have a problem with the module, i try :
 - pytest
 -python -m pytest
 
 in pycharm inside the pi_server folder: 
  - pytest ----> FAIL 
  -  python -m pytest  ----> PASSED

Reply:
both should work if it points to the same pytest :)